azkaban-webserver/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -821,6 +829,15 @@ private void ajaxExecuteFlow(HttpServletRequest req,
     ret.put("execid", exflow.getExecutionId());
   }
 
+  /* Reset flow priority if submitting user is not a Azkaban admin */
+  private void fixFlowPriorityByPermission(ExecutionOptions options, User user) {
```

Comment:
just wondering, is it better if we default to DEFAULT_FLOW_PRIORITY and only bump it up to ADMIN_PRIORITY if the submitter is an admin?

Though logically it will behave the same